ops/gulp/pkg-icons.js

```diff
@@ -0,0 +1,9 @@
+const gulp = require('gulp');
+const gulpSequence = require('gulp-sequence');
+
+const task = (cb) => {
+    gulpSequence('generateIcons','copyIconsSvg', 'copyIconsCss', cb);
```

Comment:
This task should assume that the icons files have already been built. It's job should be only related to whatever needs to happen when the package is being built and moving files from `src` to `dist`.